usecases/collect_nodes_statuses.py

```diff
@@ -29,4 +30,11 @@ def _send(self, data: dict) -> None:
             self.log.warning("collect_status_error", error=data['error'])
             return
         node = Node.from_status_dict(data)
+        # XXX: a big enough known_peers list can reach the limit of data passed to a lambda.
```

Comment:
I think we should reference here the issues that are handling this problem

Reply:
Added